search/tests/test_search_result_processor.py

```diff
@@ -264,6 +267,56 @@ def test_excerpt_back(self):
         srp = SearchResultProcessor(test_result, "dog")
         self.assertEqual(srp.excerpt[-33:], "Match upon last word - <b>Dog</b>")
 
+    @ddt.data(
+        (u'"many times"', u"And <b>many times</b> I've cried"),
+        (u'"Don\'t leave"', u"ago <b>Don't leave</b> me waiting here"),
+        (
+            u'"long and winding" leads', 
+            u'The <b>long and winding</b> road That <b>leads</b> to your door'
+        ),
+        (u'"long and winding" night', u'The <b>long and winding</b> road That leads to your door'),
+    )
+    @ddt.unpack
+    def test_excerpt_quoted(self, search_phrase, expected_excerpt):
+        test_result = {
+            "content": {
+                "notes": (
+                    "The long and winding road "
```

Comment:
1. Can you help shrink these tests?  Just include a 3-5 line note that includes what you need.
2. You don't need the line `Match upon last word - Dog`, because that was used for another test, unless you want to test an excerpt_quoted match against the end and start (like the earlier tests with this quote).
3. You could add some unicode to one of your test cases.

Also, if you take the other two tests with this same note and shrink them to 3 lines (or whatever is necessary), that would be much appreciated. :)